<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SceneSelectionWindow</class>
 <widget class="QWidget" name="SceneSelectionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="currentText">
    <string>顶板</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="maxVisibleItems">
    <number>3</number>
   </property>
   <property name="maxCount">
    <number>3</number>
   </property>
   <item>
    <property name="text">
     <string>顶板</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>斜板</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>侧板</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="conform_button">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quit_button">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
